.php-cs-fixer.php

```diff
@@ -17,6 +17,7 @@
         'native_function_invocation' => ['include' => ['@all']],
         'final_class' => true,
         'php_unit_mock_short_will_return' => true,
+        'nullable_type_declaration_for_default_null_value' => true,
```

Comment:
This rule is already part of the Symfony group, so it's currently useless.
By the way, we should update the PHP80Migration group -> PHP81Migration

Reply:
Done. Thank you for the hints!